build.gradle

```diff
@@ -11,28 +11,29 @@ buildscript {
     // changing them. Please note that, for using in Android-Components, the
     // versions below must match the ones in that repository.
     ext.versions = [
-        android_components: '5.0.0',
+        android_components: '15.0.0',
         android_gradle_plugin: '3.4.1',
         android_maven_publish_plugin: '3.6.2',
-        coroutines: '1.2.2',
+        coroutines: '1.3.0',
         dokka: '0.9.17',
         jna: '5.2.0',
         junit: '4.12',
         mockito: '2.24.5',
         mockwebserver: '3.10.0',
-        kotlin: '1.3.40',
+        kotlin: '1.3.50',
         robolectric: '4.2.1', // This is different than a-c, but we're fine, it's only tests.
         rust_android_plugin: '0.8.1',
 
         // Android X dependencies
-        androidx_annotation: '1.0.2',
-        androidx_appcompat: '1.0.0',
-        androidx_browser: '1.0.0',
-        androidx_core: '1.0.0',
+        androidx_annotation: '1.1.0',
+        androidx_appcompat: '1.1.0',
+        androidx_browser: '1.2.0-alpha07',
+        androidx_core: '1.1.0',
         androidx_espresso: '3.2.0',
         androidx_junit: '1.1.1',
+        androidx_lifecycle_extensions: '2.1.0',
         androidx_test: '1.2.0',
-        androidx_work: '2.0.1',
+        androidx_work: '2.2.0',
```

Comment:
Welcome to WorkManager 2.2, where we fixed all the old bugs and introduced new ones!

Reply:
Ahahaha, right :D Let's hope more was fixed than broken :P